pkg/auth/tokens/token_util.go

```diff
@@ -106,7 +109,7 @@ func VerifyToken(storedToken *v3.Token, tokenName, tokenKey string) (int, error)
 	if storedToken.ObjectMeta.Name != tokenName {
 		return 422, errors.New("Invalid auth token value")
 	}
-	if err := common.VerifyHash(storedToken.Token, tokenKey); err != nil {
+	if err := VerifyHash(storedToken.Token, tokenKey); err != nil {
```

Comment:
I would prefer a rename here, find by symbol is impacted since now we have 2 `VerifyHash` functions in the codebase. We should also clarify when to use each. 
```suggestion
	if err := VerifySHA256Hash(storedToken.Token, tokenKey); err != nil {
```